RealmSwift/Error.swift

```diff
@@ -119,3 +119,10 @@ public func == (lhs: Error, rhs: Error) -> Bool {
 public func ~= (lhs: Realm.Error, rhs: Error) -> Bool {
     return lhs == rhs
 }
+
+extension NSError {
```

Comment:
I could see this having use, but I still do not think it's worth polluting the NSError namespace.